### ISSUE TITLE
Consider user's locale when choosing font inside Brave

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#5434]
**Describe the issue you encountered:** On Brave you get the backslash on the URL bar, while on Brave you get the yen mark on Windows in Japanese. 

![clipboard01](https://cloud.githubusercontent.com/assets/3362943/20036302/66457668-a448-11e6-9e97-35a289208be3.png)

According to [this article on Wikipedia](https://en.wikipedia.org/wiki/Backslash#Confusion_in_usage) there are some confusions in usage. 

**Expected behavior:** Let's decide it... In any case the yen mark must be able to be used in some way.

- Platform (Win7, 8, 10? macOS? Linux distro?): Windows only

- Brave Version: 0.12.7

- Steps to reproduce:
    1. Install Windows in Japanese
    2. Install Brave

- Any related issues:

**Comment by bsclifton** [comment]
@luixxiul I believe this is because the font is MS Mincho; the character set has \ showing as yen symbol.  We should be able to solve by using a different font

**Comment by luixxiul** [comment]
@bsclifton Exactly! If you change the font-family on the URL bar in Japanese fonts like Meiryo, you get the yen mark.

**Comment by luixxiul** [comment]
So I believe this should be closed in favor of #1757.

**Comment by bsclifton** [comment]
@luixxiul I think it's a good idea to leave open :smile: We should be conscious of good default fonts depending on the user's locale :smile: I'll reopen
